src/fu-tool.c

```diff
@@ -2052,6 +2052,43 @@ fu_util_remote_enable(FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_toggle_test_mode(FuUtilPrivate *priv, gboolean enable, GError **error)
+{
+	FwupdRemote *remote = NULL;
+
+	if (!fu_util_start_engine(priv, FU_ENGINE_LOAD_FLAG_REMOTES, priv->progress, error))
+		return FALSE;
+
+	remote = fu_engine_get_remote_by_id(priv->engine, "fwupd-tests", error);
+	if (remote == NULL)
+		return FALSE;
+
+	if (!fu_engine_modify_remote(priv->engine,
+				     fwupd_remote_get_id(remote),
+				     "Enabled",
+				     enable ? "true" : "false",
+				     error))
+		return FALSE;
```

Comment:
the other idea I have for this is to special case the `fwupd-tests` remote where it's only loaded if the engine is in test mode.  Then it can default to `Enabled` in the conffile but _normally_ not be used.